Node JS/session fswd 01/UI/food-adda/package.json

```diff
@@ -1,17 +1,24 @@
 {
```

Comment:
review this package.json and check what is needed. Make absolutely sure that you do not replace the existing content